pkg/drain/nodegroup.go

```diff
@@ -143,6 +145,12 @@ func (n *NodeGroupDrainer) Drain() error {
 				if pending == 0 {
 					drainedNodes.Insert(node)
 				}
+
+				// only wait if we're not on the last node of this iteration
+				if n.nodeDrainWaitPeriod > 0 && i < newPendingNodes.Len()-1 {
+					logger.Debug("waiting for %.0f seconds before draining next node", n.nodeDrainWaitPeriod.Seconds())
+					time.Sleep(n.NodeDrainWaitPeriod)
```

Comment:
This will not compile. It should be:

```suggestion
					time.Sleep(n.nodeDrainWaitPeriod)
```